sdks/python/apache_beam/runners/portability/fn_api_runner/fn_runner.py

```diff
@@ -840,6 +841,12 @@ def _execute_bundle(self,
           # so we create a copy of the buffer for every new stage.
           runner_execution_context.pcoll_buffers[buffer_id] = buffer.copy()
           buffer = runner_execution_context.pcoll_buffers[buffer_id]
+        # When the buffer is not in the pcoll_buffers, it means that the
+        # it could be an empty PCollection. In this case, get the buffer using
+        # the buffer id and transform id
+        if buffer is None:
+          buffer = bundle_context_manager.get_buffer(buffer_id, transform_id)
+          assert buffer is not None
```

Comment:
can we log this with a clear error message?

Reply:
I was meant to remove the assertion from there. It was just there for debugging purpose.

Reply:
I see. Thanks!